docs/metrics.md

```diff
@@ -31,6 +31,7 @@ bucket.batch.objectsadded                 | meter     | number of objects added
 bucket.memory.shared                      | counter   | number of buckets referenced (excluding publish queue)
 bucket.merge-time.level-<X>               | timer     | time to merge two buckets on level <X>
 bucket.snap.merge                         | timer     | time to merge two buckets
+bucketlist.size.bytes                     | meter     | size of the BucketList in bytes, excluding future buckets
```

Comment:
I would avoid mentioning future buckets to prevent confusion, since it's more of an implementation detail (future buckets aren't actually part of the bucketlist, just an optimization we do ahead of time)